src/overlays/ovl_i6/fox_ve2.c

```diff
@@ -0,0 +1,4132 @@
+                Math_SmoothStepToF(D_801779A8, 70.0f, 1.0f, 2.0f, 0.0f);
+            }
+
+            switch (D_80177A80) {
+                case 60:
+                    for (i = 0; i < ARRAY_COUNT(gEffects); i++) {
+                        if ((gEffects[i].obj.id == OBJ_EFFECT_383) && (gEffects[i].obj.status == 2)) {
+                            Object_Kill(&gEffects[i].obj, &gEffects[i].sfxPos);
+                            break;
+                        }
+                    }
+                    break;
+                case 25:
+                    for (i = 0; i < 12; i++) {
+                        func_i6_80193668(&gObjects80[i], player->pos.x, player->pos.y,
+                                         (player->camEye.z - D_80177D20) + (2195.0f * i), i);
+                    }
+                    break;
+                case 150:
+                    Audio_PlaySfx(0x1900602A, &D_800C5D28, 0, &D_800C5D34, &D_800C5D34, &D_800C5D3C);
+                    break;
+                case 340:
+                    func_8001A55C(&player->unk_460, 0x11403071);
+                    func_8001D400(0);
+                    D_800D2F68 = 0;
+                    gBlurAlpha = 0xFF;
+                    gCurrentLevel = LEVEL_VENOM_2;
+                    D_8017827C = LEVELMODE_ALL_RANGE;
+                    gLevelMode = LEVELMODE_ALL_RANGE;
+                    player->unk_1D0 = 3;
+                    player->unk_144 = D_80177D20 = 0.0f;
+                    D_80177AB0 = D_80177A98 = 1;
+                    player->camEye.x = 1200.0f;
+                    player->camEye.z = 1200.0f;
+                    player->camEye.y = 1800.0f;
+                    D_801779A0 = 0.0f;
+                    D_801779B8 = 620.0f;
+                    D_801779C0 = 0.0f;
+                    player->unk_034 = 0.0f;
+                    player->unk_234 = 0;
+                    D_80177A48[0] = 1.0f;
+                    func_800A6148();
+                    gObjects58 = Memory_Allocate(200 * sizeof(Object_58));
+                    for (i = 0; i < 200; i++) {
+                        gObjects58[i].obj.status = 0;
+                    }
+                    func_i6_80193710();
+                    D_8017782C = LEVELMODE_ALL_RANGE;
+                    func_800A594C();
+                    // todo chain assign
+                    D_80178348 = D_80178350 = D_80178354 = 0;
+                    D_80178340 = D_80178358 = 0xFF;
+                    player->timer_1F8 = 2;
+                    D_80177A80 = 0;
+                    D_80177A48[1] = 0.0f;
+                    break;
+            }
+            break;
+        case 3:
+            if (player->timer_1F8 == 0) {
+                // todo chain assign
+                D_80178348 = D_80178350 = D_80178354 = 0;
+                D_80178340 = D_80178358 = 0;
+            } else {
+                // todo chain assign
+                D_80178348 = D_80178350 = D_80178354 = 0;
+                D_80178340 = D_80178358 = 0xFF;
```

Comment:
```suggestion
                D_80178348 = D_80178350 = D_80178354 = 0;
                D_80178340 = D_80178358 = 255;
```
These guys are all colors and alphas